src/components/ProducerList.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import { requestProducerData } from "../actions/index";
+
+class ProducerList extends React.Component {
+  componentDidMount() {
+    this.props.requestProducerData();
+  }
+  person = (x, i) =>
+
+  <table key={Math.random()*1000} style={{borderCollapse: 'collapse', width: '100%'}}>
```

Comment:
no in-line styles, give the component a class name then add the styles in a css file. this applies to here and everywhere else that you are doing in-line styles